lib/pinchflat_web/controllers/sources/source_html/index.html.heex

```diff
@@ -1,6 +1,16 @@
 <div class="mb-6 flex gap-3 flex-row items-center justify-between">
   <h2 class="text-title-md2 font-bold text-black dark:text-white">Sources</h2>
   <nav>
+    <.button color="bg-transparent" x-data="{ copied: false }" x-on:click={~s"
+          copyWithCallbacks(
+            '#{opml_feed_url(@conn)}',
+            () => copied = true,
+            () => copied = false
+          )
+        "}>
+      Copy OPML Feed
+      <span x-show="copied" x-transition.duration.150ms><.icon name="hero-check" class="ml-2 h-4 w-4" /></span>
+    </.button>
```

Comment:
Small and completely non-blocking, but how simple do you think it'd be to prevent the checkmark from shifting the `Copy OPML Feed` to the left?

Again, if the answer is anything other than "very easy" then this can wait

Reply:
Usually I would achieve this by using visibility: hidden; instead of display: none; but this is handled by alpine.js here. So I have to look into this a bit to find a good solution.